### PR TITLE
Add release field to sentry logger

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

In order to know when we break stuff.

## Changes

* Logger

## Sentry Docs: 
[Here](https://docs.sentry.io/clients/javascript/config/)

### CODE REVIEW

File: app/views/main.scala.html [hunk 1]
```diff
@@ -35,6 +35,9 @@
 	@content
 	<script src="https://polyfill.guim.co.uk/v2/polyfill.min.js?features=default,Array.prototype.find,fetch"></script>
 	<script defer src="https://js.stripe.com/v3/"></script>
+	<script type="text/javascript">
+        window.guardian.gitCommitId = @app.BuildInfo.gitCommitId
```
Comment (davidfurey) [inline app/views/main.scala.html]: This will fail if window.guardian hasn't been initialised

Comment (davidfurey) [inline app/views/main.scala.html]: Also, I think `@app.BuildInfo.gitCommitId` should be in single-quotes

Comment (svillafe) [inline app/views/main.scala.html]: good catch, fixed it

**Comment by svillafe** [comment]
@prout-bot Teamcity was failing to build master. Currently, is deploying the fixed version.

**Comment by svillafe** [comment]
It is working on PROD

**Comment by svillafe** [comment]
Tested the recurring paypal and stripe flow in PROD and are working.

**Pull request merged**